audio.py

```diff
@@ -88,3 +135,35 @@ def compute_duration(audio_path, backend = 'ffmpeg'):
 	cmd = ['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of', 'default=noprint_wrappers=1:nokey=1'
 			] if backend == 'ffmpeg' else ['soxi', '-D'] if backend == 'sox' else None
 	return float(subprocess.check_output(cmd + [audio_path]))
+
+
+def timeit(audio_path, numbers, sample_rate, mono, audio_backend, delimiter):
+	start_process_time = time.process_time_ns()
+	start_perf_counter = time.perf_counter_ns()
+	for i in range(numbers):
+		signal, sample_rate = read_audio(audio_path, sample_rate = sample_rate, mono = mono, backend = audio_backend)
```

Comment:
Please pass `normalize=False` for simplicity and removing this out of measurement. If we ever do a deeper investigation, we may also study separately int16->float32 conversion separately.

Reply:
done